tsconfig.json

```diff
@@ -1,15 +1,13 @@
 {
     "compilerOptions": {
-        "target": "es6",
+        "target": "ES2020",
         "module": "commonjs",
         "outDir": "./dist",
         "declaration": true,
         "sourceMap": true,
         "strict": true,
         "lib": [
-            "dom",
```

Comment:
While the client might run the browser, I haven't found any occurrences where removing the `dom` breaks TypeScript compilation. Removing it since to ensure portability with Node.